metricbeat/module/postgresql/docker-compose.yml

```diff
@@ -2,17 +2,18 @@ version: '2.3'
 
 services:
   postgresql:
-    image: docker.elastic.co/integrations-ci/beats-postgresql:${POSTGRESQL_VERSION:-13.2}-2
+    image: docker.elastic.co/integrations-ci/beats-postgresql:${POSTGRESQL_VERSION:-13.11}-2
     build:
       context: ./_meta
       args:
-        POSTGRESQL_VERSION: ${POSTGRESQL_VERSION:-13.2}
+        POSTGRESQL_VERSION: ${POSTGRESQL_VERSION:-13.11}
     environment:
       POSTGRES_PASSWORD: postgres
     ports:
       - 5432
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U postgres -p 5432"]
-      interval: 5s
-      timeout: 5s
+      test: ['CMD-SHELL', 'psql -h localhost -p 5432 -U postgres -c select 1 -d postgres']
```

Comment:
`select 1` behaviour is not very deterministic, does not return an output. 

Do you want to consider running `selection version()` instead?

What advantage `select 1` has over `pg_is_ready`, when `pg_is_ready` indicate the database is ready to accept the connection

Reply:
If you see the referenced links in PR description you can find more info but I'll put a short summary here:

- So there are two phases in which Postgres container starts up twice, one is initialization and then for real to which our testing suite is supposed to connect. As there was no `start_period` before and the usage of `pg_isready` resulted in HEALTHCHECK pass which might not be the case. Refer: https://github.com/Silicon-Ally/testpgx/blob/d198f78f1d1f18933750826f3f07f209c87b4a0b/testpgx.go#L184

- Also see: https://stackoverflow.com/a/74095912/5821408. With `pg_isready` we cannot even check with `pg_isready` if a db exists but with `psql` we can.

- Read: https://github.com/peter-evans/docker-compose-healthcheck/issues/16#issuecomment-1420377736 (there's also a note at the end). So I've set the `timeout`, `interval`, `start_period` properly to do the HEALTHCHECK more reliably now and psql now actually selects 1 row from an existing table (`select 1`) which confirms it is in a ready state and not in another state (like initialization state).

You can refer to the linked issues for more details.

Reply:
> select 1 behaviour is not very deterministic, does not return an output.

We are depending on the exit code of `psql` and not `select 1` as if `select 1` fails then psql exits with a non-zero exit code. It is fine to not have rows for a particular db but it is ensuring that it is able to connect to the same.

Reply:
Thanks for taking the time for explaining in a detailed manner.

Reply:
Thanks for reviewing 🚀